docs/access-management/rbac.md

```diff
@@ -7,28 +7,29 @@ products: oss-enterprise, cloud-teams
 :::
 
 ## Organization Resource Roles
 
 Permissions are scoped to the given Organization for which the user has this role, and any Workspaces within. 
 
-| Permissions           | Member |  Reader |  Editor | Admin |
-| :---------------------- | :--------: | :--------: | :--------: |:--------: |
-| **Read Organization**<br /><ul><li> Read individual organizations</li></ul> | X | X | X | X |
-| **Create Workspace**<br /><ul><li>Create new workspace within a specified organization</li><li>Delete a workspace</li></ul> | | | X | X |
-| **Update Organization**<br /><ul><li>Modify organization settings, including billing, PbA, SSO</li><li>Modify user roles within the organization</li></ul> | |  |  | X |
+| Permissions             | Member     |  Reader    | Runner |  Editor | Admin |
+| :---------------------- | :--------: | :--------: | :--------: | :--------: |:--------: |
+| **Read Organization**<br /><ul><li> Read individual organizations</li></ul> | X | X | X | X | X |
+| **Create Workspace**<br /><ul><li>Create new workspace within a specified organization</li><li>Delete a workspace</li></ul> | | | | X | X |
+| **Update Organization**<br /><ul><li>Modify organization settings, including billing, PbA, SSO</li><li>Modify user roles within the organization</li></ul> | | |  |  | X |
 
 ## Workspace Resource Roles
 Permissions are scoped to the specific Workspace in which the user has this role.
 
-| Permissions           | Reader    | Editor    | Admin    |
-| ---------------------- | :--------: | :--------:| :--------: |
-| **Read Workspace**<br /><ul><li>List the connections in a workspace</li><li>Read individual connections</li><li>Read workspace settings (data residency, users, connector versions, notification settings) </li></ul> | X | X | X |
-| **Modify Connector Settings**<br /><ul><li>Create, modify, delete  sources and destinations in a workspace</li></ul> | | X | X |
-| **Update Connection**<br /><ul><li> Start/cancel syncs</li><li>Modify a connection, including name, replication settings, normalization, DBT</li><li>Delete a connection</li><li> Create/Update/Delete connector builder connectors</li></ul> |  | X | X |
-| **Update Workspace**<br /><ul><li> Update workspace settings (data residency, users, connector versions, notification settings)</li><li> Modify workspace connector versions</li></ul> | |  | X |
+| Permissions           | Reader    | Runner | Editor    | Admin    |
+| ---------------------- | :--------: | :--------:| :--------:| :--------: |
+| **Read Workspace**<br /><ul><li>List the connections in a workspace</li><li>Read individual connections</li><li>Read workspace settings (data residency, users, connector versions, notification settings) </li></ul> | X | X | X | X |
+| **Modify Connector Settings**<br /><ul><li>Create, modify, delete  sources and destinations in a workspace</li></ul> | | | X | X |
+| **Sync Connection**<br /><ul><li>Start/cancel syncs and refreshes</li></ul> | | | X | X |
```

Comment:
<img width="1047" alt="image" src="https://github.com/user-attachments/assets/cbd1f059-20cf-4900-b4dd-8e0cd8283d48">
should include runner here i think

Reply:
Totally.  Fixed in https://github.com/airbytehq/airbyte/pull/46916/commits/089ba77387200a8e9dfa0ec71f4732c77931bec2 and re-ordered the rows so we have a cleaner shape.